angular-app/src/app/project.service.ts

```diff
@@ -64,6 +65,12 @@ export class ProjectService {
     return this.firestore.collection(COLLECTION_PROJECTS).doc<Project>(id).collection<ProjectFile>(COLLECTION_FILES).valueChanges();
   }
 
+  public updateLyricDoc(id: string, text: string): void {
+    this.firestore.collection(COLLECTION_PROJECTS).doc<Project>(id).update({
```

Comment:
Will this overwrite existing files associated with the project?

Reply:
It does not

Reply:
The only thin that is overwritten is the previous field that holds the text in the text area

Reply:
It's worth noting that we'll likely run into concurrency-type issues if multiple users edit simultaneously --- something to think about improving if we have time